randovania/games/samus_returns/exporter/joke_hints.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+JOKE_HINTS = [
+    "This hint has been sealed away.",
+    "Spidersparking can quickly cover large distances.\nTake advantage whenever possible!",
+    "The Spider Boost can be used while in Spider Ball form.\n"
```

Comment:
So you're referring to this now both as spider boost and spidersparking? Also is this supposed to be a multiline hint?

Reply:
I should probably rename all instances in rdv to Spider Boost since that's the technical official term.